src/Serializer/AbstractItemNormalizer.php

```diff
@@ -280,6 +280,9 @@ private function denormalizeRelation(string $attributeName, PropertyMetadata $pr
             } catch (ItemNotFoundException $e) {
                 throw new InvalidArgumentException($e->getMessage(), $e->getCode(), $e);
             } catch (InvalidArgumentException $e) {
+                if ('' === $value && ($propertyMetadata->isWritableLink() || $this->resourceClassResolver->isResourceClass($className))) {
```

Comment:
Well, this causes the validation problem. Because empty strings are often used as a null value for dropdown lists in web, for example. So you think, that you provided an analogue variant of null, you expect the validation error, as I described in #1070 but you getting the exception and it is a bit confusing. Also I've stacked with it in a few projects and I had to use my own fork in all of them due to frontend get unexpected exceptions instead of validation errors. And you just can't send "null" from the select. And I see in #1070 that I'm not the only one who had a problem with it.